CriticalMass/TwitterViewController.swift

```diff
@@ -50,3 +53,19 @@ class TwitterViewController: UIViewController {
         messagesTableViewController.tableView.separatorInset = UIEdgeInsets(top: 0.0, left: 73.0, bottom: 0.0, right: 0.0)
     }
 }
+
+private extension TwitterViewController {
+    func openTweet(_ tweet: Tweet) {
+        if let webURL = tweet.webURL, UIApplication.shared.canOpenURL(webURL) {
+            UIApplication.shared.open(webURL, options: [:], completionHandler: nil)
```

Comment:
Could we also add the universal link variant of the user has installed the Twitter app? Not entirely sure if that even works with the tweet link but would be great. Then we could route to the app if `canOpenUrl` is true and if not we open the Link in Safari or if it was for me could also be SafariViewController. 

This is also fine but maybe you could check if twitter app link is possible

Reply:
I rechecked on my phone and it already works like a charm 🦢
Let's merge (once the conflict is resolved)

Reply:
the web link should open the Twitter app if It's installed. So no need to add the twitter universal link.